monai/data/image_reader.py

```diff
@@ -132,6 +132,13 @@ class ITKReader(ImageReader):
     array index order will be `CDWH`.
 
     Args:
+        channel_dim: the channel dimension of the input image, default is None.
```

Comment:
Maybe let's change the args of `__init__()` to:
```py
def __init__(self, channel_dim: Optional[int] = None, series_name: Optional[str] = None, **kwargs):
```
Then no need to `pop` from kwargs.

Thanks.